app/api/auth/login/gallery/route.ts

```diff
@@ -1,5 +1,9 @@
-import { ConflictError } from "@/custom/errors/dictionary/errorDictionary";
+import {
+  ConflictError,
+  TooManyRequestError,
```

Comment:
Can we change this from "TooManyRequestError" to "RateLimitExceededErrror" in all instances of it's usage? Just so it's more in line with conventional naming patterns

Reply:
Okay